docs/ruleset_functions/is_baseline_system_1(HVAC_id).md

```diff
@@ -0,0 +1,51 @@
+6. is_hvac_sys_fluid_loop_purchased_CHW()
+7. is_hvac_sys_cooling_type_fluid_loop()
+8. are_all_terminal_heat_sources_none_or_null()  
+9. are_all_terminal_cool_sources_none_or_null() 
+10. are_all_terminal_fans_null()  
+11. are_all_terminal_types_CAV()  
+12. are_all_terminal_types_four_pipe()  
+13. are_all_terminal_supplies_ducted()  
+14. are_all_terminal_heat_sources_hot_water()  
+15. are_all_terminal_cool_sources_chilled_water() 
+16. does_each_zone_have_only_one_terminal()    
+17. does_hvac_system_serve_single_zone()  
+ 
+## Logic:    
+- Create an object associated with the hvac system: `hvac_b = hvac_b.id`  
+- Set is_baseline_system_1 = Not_Sys_1: `is_baseline_system_1 = "Not_Sys_1"`    
+- Check that there is no preheat system (it equals Null), if there is none then carry on: `if Len(hvac_b.preheat_system) == Null:`  
+    - Check if heatingsystem is a fluid_loop, if it is then carry on: `if is_hvac_sys_heating_type_fluid_loop(B_RMR, hvac_b.id) == TRUE:`     
+        - Check if fansystem is constant volume, if yes then carry on: `if is_hvac_sys_fan_sys_CV(B_RMR, hvac_b.id) == TRUE:`  
+            - Check if the hvac system serves a single zone and that the zone only has one terminal unit: `if does_hvac_system_serve_single_zone(B_RMR, zone_id_list) == TRUE AND does_each_zone_have_only_one_terminal(B_RMR,zone_id_list) == TRUE:`     
+                - Check that the data elements associated with the terminal unit align with system 1: `if are_all_terminal_heat_sources_none_or_null(B_RMR,terminal_unit_id_list) == TRUE AND are_all_terminal_cool_sources_none_or_null(B_RMR,terminal_unit_id_list) == TRUE And are_all_terminal_fans_null(B_RMR,terminal_unit_id_list) == TRUE AND are_all_terminal_supplies_ducted(B_RMR,terminal_unit_id_list) == FALSE:`        
+                    - if coolingsystem is DX and the heating fluid loop serves a boiler and the terminal type is constant volume then SYS-1: `if is_hvac_sys_cooling_type_DX(B_RMR, hvac_b.id) == TRUE AND is_hvac_sys_fluid_loop_attached_to_boiler(B_RMR, hvac_b.id) == TRUE AND are_all_terminal_types_CAV(B_RMR,terminal_unit_id_list) == TRUE: is_baseline_system_1 = "SYS-1"`
+                    - elif coolingsystem is DX and the fluid loop is purchased heating and the terminal type is constant volume then SYS-1b: `elif is_hvac_sys_cooling_type_DX(B_RMR, hvac_b.id) == TRUE AND is_hvac_sys_fluid_loop_purchased_heating(B_RMR, hvac_b.id) == TRUE AND are_all_terminal_types_CAV(B_RMR,terminal_unit_id_list) == TRUE: is_baseline_system_1 = "SYS-1b"`
+                    - elif the cooling system is a fluid loop and terminal type is four pipe with chilled and hot water coils: `elif is_hvac_sys_cooling_type_fluid_loop(B_RMR, hvac_b.id) == TRUE AND are_all_terminal_types_four_pipe(B_RMR,terminal_unit_id_list) == TRUE AND are_all_terminal_heat_sources_hot_water(B_RMR,terminal_unit_id_list) == TRUE AND are_all_terminal_cool_sources_chilled_water(B_RMR,terminal_unit_id_list) == TRUE:`   
```

Comment:
May need to update pending discussion of where the purchased heating and cooling coils would be defined.